include/engine/datafacade/datafacade_base.hpp

```diff
@@ -143,6 +143,8 @@ class BaseDataFacade
     virtual std::size_t GetCoreSize() const = 0;
 
     virtual std::string GetTimestamp() const = 0;
+
+    virtual bool GetUTurnsDefault() const = 0;
```

Comment:
Implement dummy for mock facade! On my unit-tests-for-v5 branch I had to hunt down a ton of extra member functions as no one cared to implement them for too long!